arkane/kinetics.py

```diff
@@ -263,11 +264,20 @@ def write_output(self, output_directory):
             f.write('#   ======= ============ =========== ============ ============= =========\n')
             f.write('\n\n')
 
-            kinetics_0_rev = Arrhenius().fit_to_data(t_list, np.array(k0_revs), kunits=self.k_r_units)
-            kinetics_rev = Arrhenius().fit_to_data(t_list, np.array(k_revs), kunits=self.k_r_units)
+            kinetics_0_rev = Arrhenius().fit_to_data(t_list, np.array(k0_revs), kunits=self.k_r_units,
+                                                     three_params=self.three_params)
+            kinetics_rev = Arrhenius().fit_to_data(t_list, np.array(k_revs), kunits=self.k_r_units,
+                                                   three_params=self.three_params)
 
             f.write('# k_rev (TST) = {0} \n'.format(kinetics_0_rev))
             f.write('# k_rev (TST+T) = {0} \n\n'.format(kinetics_rev))
+
+        if self.three_params:
+            f.write('# kinetics fitted using the modified three-parameter Arrhenius equation '
+                    'k = A * exp(-Ea/RT) * (T/T0)^n \n')
```

Comment:
usually `T^n` comes before `exp(-Ea/RT)`
(at least in the convention I'm familiar with)
It's minor, but will make it be more easily recognizable

Also, no need for the space at the end of the line (here and below)

Reply:
I moved T^n upfront. The new line is needed. Otherwise, the `rxn_str` would be on the same line as this message.

Reply:
The new line is indeed needed, but there's also a white space before it